node/sc/vt_recovery_test.go

```diff
@@ -439,6 +439,110 @@ func TestScenarioAutomaticRecovery(t *testing.T) {
 	assert.Equal(t, vtr.service2mainHint.requestNonce, vtr.service2mainHint.handleNonce)
 }
 
+// TestMultiOperatorRequestRecovery tests value transfer recovery for the multi-operator.
+func TestMultiOperatorRequestRecovery(t *testing.T) {
+	tempDir := os.TempDir() + "sc"
+	os.MkdirAll(tempDir, os.ModePerm)
+	defer func() {
+		if err := os.RemoveAll(tempDir); err != nil {
+			t.Fatalf("fail to delete file %v", err)
+		}
+	}()
+
+	// 1. Init dummy chain and do some value transfers.
+	info := prepare(t, func(info *testInfo) {
+		for i := 0; i < testTxCount; i++ {
+			ops[KLAY].request(info, info.localInfo)
+		}
+	})
+
+	// 2. Set multi-operator.
+	cAcc := info.nodeAuth
+	pAcc := info.chainAuth
+	opts := &bind.TransactOpts{From: cAcc.From, Signer: cAcc.Signer, GasLimit: testGasLimit}
+	_, err := info.localInfo.bridge.RegisterOperator(opts, pAcc.From)
```

Comment:
for local, cAcc?
for remote, pAcc?

Reply:
It's an intentional account mix to register different operators.